src/rgw/rgw_user.cc

```diff
@@ -855,7 +855,9 @@ int RGWAccessKeyPool::execute_remove(const DoutPrefixProvider *dpp,
   map<std::string, RGWAccessKey> *keys_map;
 
   if (!op_state.has_existing_key()) {
-    set_err_msg(err_msg, "unable to find access key");
+    std::string key_type_str = key_type ? "s3" : "swift";
```

Comment:
This is an enum, not a boolean, so the boolean compare isn't correct.  Probably a better solution is a little helper function to convert an ObjectKeyType into a string.

Reply:
@dang   Thanks. I have added  a conversion function, by the way, this bug can be reproduced by executing "radosgw-admin key rm --uid ... --subuser ... --access-key ..."